apps/livreGUI/tfEditor/TransferFunctionEditor.cpp

```diff
@@ -275,24 +276,42 @@ void TransferFunctionEditor::_load()
 void TransferFunctionEditor::_save()
 {
     QString filename = QFileDialog::getSaveFileName( this, "Save transfer function",
-                                                           QString(),
-                                                           TF_FILE_FILTER );
+                                                     QString(),
+                                                     TF_FILE_FILTER );
     if( filename.isEmpty( ))
         return;
 
     if( !filename.endsWith( ".tf" ))
         filename.append( ".tf" );
 
+    // save .tf transfer function file (loadable via GUI)
     QFile file( filename );
     file.open( QIODevice::WriteOnly );
-    QDataStream out( &file );
-    out.setVersion( QDataStream::Qt_5_0 );
-
-    out << TF_FILE_HEADER << TF_FILE_VERSION;
-    out << _redWidget->getPoints()
-        << _greenWidget->getPoints()
-        << _blueWidget->getPoints()
-        << _alphaWidget->getPoints();
+    QDataStream outTf( &file );
+    outTf.setVersion( QDataStream::Qt_5_0 );
+
+    outTf << TF_FILE_HEADER << TF_FILE_VERSION;
+    outTf << _redWidget->getPoints()
+          << _greenWidget->getPoints()
+          << _blueWidget->getPoints()
+          << _alphaWidget->getPoints();
+
```

Comment:
They are different in terms of how they are kept. There may be people still using the old files. We can deprecate in the next release. So, dont remove it yet, but it should be removed in long term.